src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertsArtworks.tsx

```diff
@@ -0,0 +1,317 @@
+import {
+  Text,
+  Flex,
+  Box,
+  Spacer,
+  Join,
+  Separator,
+  Button,
+  GridColumns,
+  Column,
+  SkeletonText,
+  Skeleton,
+  SkeletonBox,
+  Message,
+} from "@artsy/palette"
+import { RouterLink } from "System/Router/RouterLink"
+import { SavedSearchAlertsArtworksQuery } from "__generated__/SavedSearchAlertsArtworksQuery.graphql"
+import { createFragmentContainer, graphql } from "react-relay"
+import { SystemQueryRenderer } from "System/Relay/SystemQueryRenderer"
+import { EditAlertEntity } from "Apps/Settings/Routes/SavedSearchAlerts/types"
+import { SearchCriteriaAttributes } from "Components/SavedSearchAlert/types"
+import { getAllowedSearchCriteria } from "Components/SavedSearchAlert/Utils/savedSearchCriteria"
+import { AlertProvider } from "Components/Alert/AlertProvider"
+import ArtworkGrid, {
+  ArtworkGridPlaceholder,
+} from "Components/ArtworkGrid/ArtworkGrid"
+import { Modal } from "Components/Alert/Components/Modal/Modal"
+import { Media } from "Utils/Responsive"
+import {
+  CriteriaPills,
+  CriteriaPillsPlaceholder,
+} from "Components/Alert/Components/CriteriaPills"
+import { ModalHeader } from "Components/Alert/Components/Modal/ModalHeader"
+
+interface AlertArtworksProps {
+  alert: NonNullable<SavedSearchAlertsArtworksQuery["response"]["me"]>["alert"]
+  onCloseClick?: () => void
+  onEditAlertClick: () => void
+}
+
+export const AlertArtworks: React.FC<AlertArtworksProps> = ({
+  alert,
+  onCloseClick,
+  onEditAlertClick,
+}) => {
+  if (!alert || !alert.artworksConnection)
+    return <SavedSearchAlertsArtworksPlaseholder onCloseClick={onCloseClick} />
+
+  const count = alert.artworksConnection?.counts?.total
+  const href = alert.href
+
+  return (
+    <>
+      <Media greaterThanOrEqual="md">
```

Comment:
**nitpick / non-blocking**: there are a few duplicated parts inside (buttons, messages, pills), I think it's possible to extract them to a common place, but up to you